src/features/workflow/Workflow.tsx

```diff
@@ -57,7 +57,7 @@ const Workflow: React.FC<WorkflowProps> = ({ qriRef }) => {
       dispatch(showModal(ModalType.workflowSplash))
     }
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [location])
```

Comment:
ah yes this makes sense. we'd need to destructure the location object for this to work properly. We can deal with it later in an audit of hook deps